packages/medusa/src/models/customer.ts

```diff
@@ -20,6 +20,8 @@ import { generateEntityId } from "../utils/generate-entity-id"
 
 @Entity()
 @Unique(["email", "has_account"])
+@Index("idx_customer_first_last_phone", ["first_name", "last_name", "phone"])
```

Comment:
Why do we have an index for these fields?

Reply:
Nice catch, i need to remove this one I thought I did it 👍

Reply:
todo: